src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22450.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue22450 : _IssuesUITest
+	{
+		public override string Issue => "[Android] Custom shell back button is always white in color";
+
+		public Issue22450(TestDevice device) : base(device) { }
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		[FailsOnWindows("The golden tests cut the title bar where the back button appears.")]
```

Comment:
This attribute has changed:
```
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22450.cs(16,4): error CS0246: The type or namespace name 'FailsOnWindowsAttribute' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.Android.Tests/Controls.TestCases.Android.Tests.csproj]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22450.cs(16,4): error CS0246: The type or namespace name 'FailsOnWindows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.Android.Tests/Controls.TestCases.Android.Tests.csproj]
    0 Warning(s)
    2 Error(s)
```

Could you update it, and use `FailsOnWindowsWhenRunningOnXamarinUITest`?